package.json

```diff
@@ -15,20 +15,24 @@
         "lint:text": "d2-style text check",
         "start": "d2-app-scripts start",
         "test": "d2-app-scripts test",
-        "cy:start": "BROWSER=none yarn start",
-        "cy:open": "d2-utils-cypress open --appStart 'yarn cy:start'",
-        "cy:run": "d2-utils-cypress run --appStart 'yarn cy:start'"
+        "cypress:start": "BROWSER=none REACT_APP_DHIS2_BASE_URL=https://debug.dhis2.org/dev REACT_APP_DHIS2_API_VERSION=37 yarn start",
+        "cypress:open": "d2-utils-cypress open --appStart 'yarn cypress:start'",
+        "cypress:run": "d2-utils-cypress run --appStart 'yarn cypress:start'",
+        "cypress:capture": "d2-utils-cypress run --capture --appStart  'yarn cypress:start' --serverMinorVersion 37",
+        "cypress:stub": "d2-utils-cypress run --stub --appStart  'yarn cypress:start' --serverMinorVersion 37"
```

Comment:
These need to be adjusted, `--appStart`  has been removed.

@ismay used a helper tool from one of the Cypress devs in scheduler: https://github.com/dhis2/scheduler-app/blob/master/package.json#L14-L16

You may also need to adjust the e2e job in the workflow: ref https://github.com/dhis2/scheduler-app/blob/master/.github/workflows/dhis2-verify-app.yml#L66-L92